internal/metamorph/processor.go

```diff
@@ -298,7 +298,7 @@ func (p *Processor) StartProcessMinedCallbacks() {
 }
 
 func (p *Processor) updateMined(ctx context.Context, txsBlocks []*blocktx_api.TransactionBlock) {
-	_, span := tracing.StartTracing(ctx, "updateMined", p.tracingEnabled, p.tracingAttributes...)
+	ctx, span := tracing.StartTracing(ctx, "updateMined", p.tracingEnabled, p.tracingAttributes...)
 	defer tracing.EndTracing(span)
 
 	updatedData, err := p.store.UpdateMined(p.ctx, txsBlocks)
```

Comment:
We need to pass here `ctx` generated 3 lines above instead of `p.ctx` to have a continuous trace or it doesn't matter?

Reply:
To the `store.UpdateMined` function, just to clarify ☝️

Reply:
Oh, good catch 👍 

It should be `ctx` not `p.ctx`